pubspec.yaml

```diff
@@ -1,13 +1,13 @@
-name: uri
```

Comment:
sorry about the reordering noise here - side effect of using jefe to manage my project dependencies. I'll tidy this up once we get to the code being in a shape you are happy with